rickynyairo/StackOverflow-lite
# Chore: Add badges and separate the app into modules

#### What does this PR do?
Adds badges to README file 
Separates app entities into their different modules
Incorporates blueprints for the app

#### Description of Task to be completed
The badges give information about the API to any user who would be interested.
This ensures that functionality does not clash between entities.

#### How should this be manually tested?
Clone the repository [here](https://github.com/rickynyairo/StackOverflow-lite/)

>git clone https://github.com/rickynyairo/StackOverflow-lite/

Navigate to the API directory
install the requirements (this should be done in a virtual environment)

> pip install requirements.txt 

While in the virtual environment (in the API directory), run the following command to perform the tests:

>nosetests app/tests -v

#### Background context
The API defines a set of endpoints for the StackOverflow-Lite project

#### What are the relevant pivotal tracker stories?
[#159831786](https://www.pivotaltracker.com/story/show/159831786)
